website/docs/d/org_user.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_org_user"
+sidebar_current: "docs-vcd-datasource-org-user"
+description: |-
+  Provides a vCloud Director Organization user data source. This can be used to read organization users.
+---
+
+# vcd\_org\_user
+
+Provides a vCloud Director Org User data source. This can be used to read organization users, including org administrators.
+
+Supported in provider *v2.10+*
```

Comment:
Please change from `*v2.10+*` to `*v3.0+*` everywhere, as we have made the decision to name next release 3.0.

Reply:
done. Also changed /v2 to /v3 in go.mod